polkadot/node/subsystem-bench/src/lib/network.rs

```diff
@@ -548,6 +566,10 @@ async fn emulated_peer_loop(
 					emulated_peer.rx_limiter().reap(size).await;
 					stats.inc_received(size);
 
+					if peer_message.is_from_node() && emulated_peer.has_latency() {
+						emulated_peer.emulate_latency().await;
```

Comment:
This will effectively stall the peer loop for up to whatever latency/2 is configured, meaning the peer also won't send messages to the node, even if we request it to from main thread. This latency needs to be emulated in the task that is supposed to send messages from the node to the peer, see `proxy_send_request` method.

Reply:
Changed it. Is that what you were thinking about?